### DISCUSSION TITLE
Is studio here?

### DISCUSSION THREAD

**Discussion by MarcosGraficon** [OCA/odoo-community.org#143] [Q&A] [answered]
Hey quick question is there the odoo studio app made in OCA? if it is whats the name?
![image](https://github.com/OCA/odoo-community.org/assets/156304478/26122710-7d99-4217-bd96-fad74da066c1)

**Comment by vdewulf** [comment, marked as answer]
Hello,

There is no studio App in the OCA.

However, you might find a lot of useful modules in the thousands of existing modules that might do what you need.

There is also a module "module web_custom_modifier" that's being proposed currently to the OCA by Numigi team:
https://github.com/OCA/web/pull/2707

Have a nice day!
